flytepropeller/pkg/apis/flyteworkflow/v1alpha1/node_status.go

```diff
@@ -30,7 +31,7 @@ func (in *MutableStruct) ResetDirty() {
 	in.isDirty = false
 }
 
-func (in MutableStruct) IsDirty() bool {
+func (in *MutableStruct) IsDirty() bool {
```

Comment:
There were misc warnings about mixing pointer and non-pointer receiver functions so I fixed that up.